doc/en/Specialist_tools/JSXGraph/Author_quickstart.md

```diff
@@ -14,3 +14,194 @@ __We strongly recommend you do not use an HTML-aware editor when editing STACK q
+
+## Feedback
+
+In general, when comparing floating point values (such as points or values derived their coordinates), it is advisable to use a numerical answer test, due to rounding errors intrinsic to floating point numbers. For example, you might have noticed that even with the snap to grid option, the values in the input field looked like `[0.8,1.2000000000000002]` rather than `[0.8,1.2]`.
+
+For our question, if there was a unique correct answer, we could simply compare the student answer `ans1` to our teacher answer `ta1`, both being a point (i.e., a list of two coordinates), using an answer test like `NumAbsolute` or `NumRelative`.
+
+However, as there are many possible correct answers, we will compute the slope of the student's line and compare it to the slope of our function.
+
+So in the feedback variables of PRT1, we define
+
+```
+student_slope: ans1[2]/ans1[1];
+```
+
+and in the PRT1, we compare `student_slope` and `slope` using the answer test `NumRelative` with _Test Options_ 0.02 and _Quiet_ set to _Yes_.
+
+### JSXGraph in feedback
+
+JSXGraph plots can not only be used in the question text, but also in the feedback. For example, in our question, if the student gets the answer incorrect, we could display a plot showing the function f(x) as well as the student answer, for comparsion. Add the following to the false feedback for PRT1:
+
+```
+<p>The slope of your line is different from the slope of {@expr@}, see below:</p>
+
+[[jsxgraph]]
+var board = JXG.JSXGraph.initBoard(divid, {
+    axis: true,
+    showCopyright: false,
+    showNavigation: false,
+    boundingbox: [-6, 6, 6, -6],
+    keepaspectratio: true,
+    grid:true
+});
+
+var pO = board.create('point', [0, 0], {name: 'O', fixed: true, color: "blue" });
+var pA = board.create('point', {# ans1 #}, { name: 'A',  fixed: true});
+var line = board.create('line', [pO, pA], { fixed: true });
+var graph = board.create('functiongraph', '{#expr#}' , {name:'{#expr#}', withLabel: true});
+[[/jsxgraph]]
+```
+
+Note how we are using the student answer `ans1` in our plot, and we're drawing the graph of f(x) using a `functiongraph` object. Because our function is simple, its maxima string representation works as a definition for the `functiongraph`, but in general read [Basic_plots](Basic_plots.md) to see how to plot functions.
+
+### Blank inputs
+
+Note that if the student does not move the point, the input field will remain blank and they will not get any feedback. In our question, we made sure in our randomization that the initial position of A is not a correct answer, but this can be a trickier issue in particular with questions that have multiple points as inputs, as all of them may need to be moved for feedback to trigger. 
```

Comment:
If you have a situation where feedback triggering depends on multiple bound inputs, and one might not actually move all those bound things. Try `stack_jxg.define_group([p1,p2,p3,...]);` after you have bound those points (or sliders), this will make it so that if any of that group moves, all of them are considered moved and the inputs will get values. Those moves do not propagate over groups, i.e., if there are groups `{p1,p2}` and `{p2,p3}` moving `p1` will lead to `p2` being considered moved but not `p3` however moving `p2` will mark both `p1` and `p3` as moved.

Reply:
Thanks @geoo89 and @aharjula much appreciated!